.scripts/build_steps.sh

```diff
@@ -54,12 +54,6 @@ if [[ -f "${FEEDSTOCK_ROOT}/LICENSE.txt" ]]; then
   cp "${FEEDSTOCK_ROOT}/LICENSE.txt" "${RECIPE_ROOT}/recipe-scripts-license.txt"
 fi
 
-if [[ "${sha:-}" == "" ]]; then
-  pushd ${FEEDSTOCK_ROOT}
-  sha=$(git rev-parse HEAD)
-  popd
-fi
```

Comment:
context for moving this line?

Reply:
This was done by the bot, see this commit:
https://github.com/conda-forge/usgscsm-feedstock/pull/22/commits/fb2ba1abb37cdf0f5be098c719c0cdd8ea6c061e